storm-core/test/clj/backtype/storm/security/auth/ThriftClient_test.clj

```diff
@@ -20,26 +20,26 @@
   (:import [org.apache.thrift.transport TTransportException])
 )
 
+(def TIMEOUT (Integer. (* 3 1000)))
```

Comment:
How about adding a comment describing why the choice of 3000 in this particular test?